### PR TITLE
docs(samples): fix typo

### PR DESCRIPTION
**Pull request by stephaniewang526**

### CODE REVIEW

File: samples/snippets/src/main/java/com/example/bigquery/ExportQueryResultsToS3.java [hunk 1]
```diff
@@ -43,7 +43,8 @@ public static void main(String[] args) throws InterruptedException {
     // Export result of query to find states starting with 'W'
     String query =
         String.format(
-            "EXPORT DATA WITH CONNECTION %s AS SELECT * FROM %s.%s.%s WHERE name LIKE 'W%%'",
+            "EXPORT DATA WITH CONNECTION `%s` OPTIONS(uri='%s', format='%s') "
+              + "AS SELECT * FROM %s.%s.%s WHERE name LIKE 'W%%'",
             connectionName, destinationUri, format, projectId, datasetName, externalTableName);
```
Comment (lesv) [inline samples/snippets/src/main/java/com/example/bigquery/ExportQueryResultsToS3.java:48]: It looks like you are adding at least two parameters, but not actually adding them.

**Review by lesv — CHANGES_REQUESTED** [review]

File: samples/snippets/src/main/java/com/example/bigquery/ExportQueryResultsToS3.java (hunk 1 above)
Comment (stephaniewang526) [reply samples/snippets/src/main/java/com/example/bigquery/ExportQueryResultsToS3.java:48]: they were missing before!

**Review by lesv — APPROVED** [review]

File: samples/snippets/src/main/java/com/example/bigquery/ExportQueryResultsToS3.java (hunk 1 above)
Comment (lesv) [reply samples/snippets/src/main/java/com/example/bigquery/ExportQueryResultsToS3.java:48]: LOL - I should have counted earlier - sorry.

Comment (stephaniewang526) [reply samples/snippets/src/main/java/com/example/bigquery/ExportQueryResultsToS3.java:48]: No thank you for paying attention! Appreciate it :)

**Pull request merged**